web/packages/teleport/src/Navigation/Navigation.tsx

```diff
@@ -123,6 +125,100 @@ export function Navigation() {
   return (
     <NavigationContainer>
       <CategoriesContainer>{categories}</CategoriesContainer>
+      {cfg.edition === 'oss' && <AGPLFooter />}
+      {cfg.edition === 'community' && <CommunityFooter />}
     </NavigationContainer>
   );
 }
+function AGPLFooter() {
+  return (
+    <LicenseFooter
+      title="AGPL Edition"
+      subText="Unofficial Version"
+      infoContent={
+        <>
+          This is an independently compiled AGPL-3.0 version of Teleport. You
+          can find the official release{' '}
+          <Text
+            as="a"
+            href="https://goteleport.com/download/?utm_source=oss&utm_medium=in-product&utm_campaign=limited-features"
+            target="_blank"
+          >
+            here.
```

Comment:
Could we use some variation of this? I think links look much more professional when they don't use "here" as an anchor.

https://www.smashingmagazine.com/2012/06/links-should-never-say-click-here/

![example](https://github.com/gravitational/teleport/assets/27113/e98db9a3-8dd6-49e7-bab9-73003f089eab)

<details>
<summary>Patch</summary>

The other footers would have to be broken up manually with `<br/>` as well to account for `width: max-content;`

```diff
diff --git a/web/packages/teleport/src/Navigation/Navigation.tsx b/web/packages/teleport/src/Navigation/Navigation.tsx
index 0742af8aec..1b645f3976 100644
--- a/web/packages/teleport/src/Navigation/Navigation.tsx
+++ b/web/packages/teleport/src/Navigation/Navigation.tsx
@@ -137,15 +137,19 @@ function AGPLFooter() {
       subText="Unofficial Version"
       infoContent={
         <>
-          This is an independently compiled AGPL-3.0 version of Teleport. You
-          can find the official release{' '}
+          {/* This is an independently compiled AGPL-3.0 version of Teleport. You */}
+          {/* can find the official release on{' '} */}
+          This is an independently compiled AGPL-3.0 version of Teleport.
+          <br />
+          Visit{' '}
           <Text
             as="a"
             href="https://goteleport.com/download/?utm_source=oss&utm_medium=in-product&utm_campaign=limited-features"
             target="_blank"
           >
-            here.
-          </Text>
+            the Downloads page
+          </Text>{' '}
+          for the official release.
         </>
       }
     />
@@ -209,7 +213,7 @@ const SubText = styled(Text)`
 `;
 
 const TooltipContent = styled(Box)`
-  width: 360px;
+  width: max-content;
   position: absolute;
   bottom: 0;
   left: 24px;

```
</details>